<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="lay" stretch="1,2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <property name="formAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Угол</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="axisLine">
          <property name="text">
           <string>45</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Энергия</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="energyLine">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>300</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Масса</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="massLine">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Коэфф.</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="kLine">
          <property name="text">
           <string>0.5</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>g</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="gLine">
          <property name="text">
           <string>9.81</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Шаг</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="stepLine">
          <property name="text">
           <string>0.001</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QPushButton" name="buildPlot">
          <property name="text">
           <string>Построить</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QComboBox" name="modeBox">
          <item>
           <property name="text">
            <string>Аналитический способ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Рунге-Кутты</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Аддамс</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QPushButton" name="buildAxisPlot">
          <property name="text">
           <string>Построить от углов</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Начало</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="startLineEdit">
              <property name="text">
               <string>0.001</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Конец</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="endLineEdit">
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="10" column="1">
         <widget class="QPushButton" name="errorButton">
          <property name="text">
           <string>Построить график ошибки</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>От</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="fromLine">
                <property name="text">
                 <string>-100</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>До</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="toLine">
                <property name="text">
                 <string>100</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Кол-во массовых запросов</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="massReverseLine">
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="11" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QPushButton" name="reverseButton">
            <property name="text">
             <string>Обратная задача</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="massReverseButton">
            <property name="text">
             <string>Массовый запрос</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vertLay">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1098</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
